### ISSUE TITLE
[Suggestion] Add Baubles Support to Jetpacks

### ISSUE THREAD

**Issue by Endgineer** [CyberdyneCC/SimplyJetpacks-2#94]
The tiered Flux Capacitors and Reservoirs from Thermal Expansion seem to have built-in Baubles support. So I was thinking...

It would probably be a really nice solution to #28 or issues of that sort, in a sense that it would allow players to equip their special armor sets in the regular armor slots while still retaining the ability to fly. Essentially, the jetpacks could be registered as a body bauble, allowing it to be equipped in the body bauble slot- in turn, allowing players to use their armor slots for other things.

I hope you consider this suggestion; can't live without Simply Jetpacks 2. :3

**Comment by thundr** [comment]
This would make the entire line of armored jetpacks and the jetplates pointless, and I'm not sure that any of the bauble slots fit the spirit of the jetpack. IIRC there is amulet, rings, belt, and cloak and none of those would make sense for the jetpacks. 

Tomson has the final word, but I don't think it makes a lot of sense.

**Comment by Tomson124** [comment]
Exactly what I would say, too thundR :) Don't think it fit in those slots as Baubels is built arround magic themed mods and SJ2 is a addon for technic mods. There will be an option to integrate it in some armors in the future (like the one from EnderIO) but Baubles support is not a good idea in my opinion.

**Comment by ChromaPIE** [comment]
I was thinking that could the jetpacks be **attached** to the armors? You may heard of the Radiation Shielding in the mod NuclearCraft. It supports almost all of the items that could be equipped as armors simply through the crafting grid. Maybe attaching/unloading the jetpacks to the armors could be a future plan I think. After all lots of players simply wants the joyful flying experience the jetpacks brought **and** some cool skills that other armors have, no matter what methods it takes.

**Comment by GenericQuinn** [comment]
I agree with ChromaPIE. I'm looking at jetpacks in FTB Rev and my choices are, ditch my tinkers chestplate in favor of one of your jetpacks, use a flying ring (Focusing on tech, not magic), or use the outdated IC2 Jetpack which I could attach with tape. I don't know how attaching an IC2 electric jetpack to my tinkers armour would go but I just want to use simply jetpacks and keep my chestplate. A Mandarorian has no issue with slapping a jetpack onto their back and keeping their armour. 

TLDR: Please make the armour attachable and detachable to a chest plate so unique armour isn't unusable because of our desire to use one of your awesome jetpacks!
